src/ui/hooks/recordings.ts

```diff
@@ -354,21 +354,9 @@ export function useUpdateIsPrivate() {
 
 export function useIsOwner(recordingId: RecordingId) {
   const { userId } = useGetUserId();
-  const { data, loading, error } = useQuery(
-    gql`
-      query GetOwnerUserId($recordingId: UUID!) {
-        recording(uuid: $recordingId) {
-          uuid
-          owner {
-            id
-          }
-        }
-      }
-    `,
-    {
-      variables: { recordingId },
-    }
-  );
+  const { data, loading, error } = useQuery(GET_RECORDING_USER_ID, {
+    variables: { recordingId },
+  });
```

Comment:
I replaced this query with GET_RECORDING_USER_ID because the contents of these two queries are the same except for the query name itself, but I'm not 100% sure this change is correct.  Do these query names actually matter?

Reply:
This should be okay. The name matters for refetching after a query, since the query name's used to figure out what to refetch. For example, we refetch `GetRecording` here after the query to initialize a recording runs: https://github.com/RecordReplay/devtools/blob/master/src/ui/hooks/recordings.ts#L712

Both `GetOwnerUserId` and `GetRecordingUserId` aren't being refetched so this should be good as is.